WHATS_NEW.md

```diff
@@ -5,8 +5,12 @@ For information on what was included in latest release please refer to our [chan
 
 ## 0.50.0
 
-### Full support for Redhat Openshit Connectors (RHOC)
-While RHOC itegration was added all new commands are now all visable. You can create a new connector by using `rhoas connector build` in conjunction with `rhoas connector create`.
+### Support for Red Hat Openshift Connectors (RHOC)
+Our expanded support for RHOC is now visable for you to use. You can create a new connector by using `rhoas connector build` in conjunction with `rhoas connector create`.
+
```

Comment:
Should be "Red Hat OpenShift" :)

Also "Our expanded support for Red Hat OpenShift Connectors is now visible".  (RHOC acronym is internal only, fixes "visable" typo)

Could also use "available" instead of "visible"?